src/applications/disability-benefits/all-claims/tests/pages/prisonerOfWar.unit.spec.jsx

```diff
@@ -16,13 +16,19 @@ describe('Prisoner of war info', () => {
     uiSchema,
   } = formConfig.chapters.disabilities.pages.prisonerOfWar;
 
+  const formData = {
```

Comment:
I realize that you're testing the `isWithinRange` function separately, but feels as though we should also maybe add a couple of tests into the POW page to ensure that the validation is being applied to the UI correctly. I.e., we get error(s) on the page, and can't continue.